### DISCUSSION TITLE
Do I need to run the code on the terminal everytime I need to start the bot?

### DISCUSSION THREAD

**Discussion by Luttudo** [codetheweb/muse#1003] [Q&A]
Do i need to run this everytime?

`docker run -it -v "$(pwd)/data":/data -e DISCORD_TOKEN='' -e SPOTIFY_CLIENT_ID='' -e SPOTIFY_CLIENT_SECRET='' -e YOUTUBE_API_KEY='' codetheweb/muse:latest`

how to I start the bot automatically when the computer starts?

**Comment by codetheweb** [comment]
I think you can add `--restart always` to start on boot.
